src/QAT/qat/purr/compiler/execution.py

```diff
@@ -465,7 +465,7 @@ def build_pulse_channel_buffers(
     def process_pulse(
         self,
         position: PositionData,
-        buffer: List[float],
```

Comment:
We don't enforce numpy array usage anywhere, the original was correct. Please change back.